dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp

```diff
@@ -601,6 +601,7 @@
 
                 var customEvent = document.createEvent("CustomEvent");
                 customEvent.initCustomEvent("ng-event", false, false,  {
+                    htmlUrl: data["htmlPageReferer"],
```

Comment:
Can't do this, this event should "close" the dialog, we need an event that does what we need to do, we're mixing things here.